examples/RadioButton/e2e/test_radio_button.py

```diff
@@ -20,6 +21,7 @@ def go_to_app(page: Page, streamlit_app: StreamlitRunner):
     page.goto(streamlit_app.server_url)
     # Wait for app to load
     page.get_by_role("img", name="Running...").is_hidden()
+    yield
```

Comment:
Do we need this? Tests pass for me locally without this

Reply:
I'm removing this. Thanks